### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by FireTonnor**
Added mov-cli

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ https://user-images.githubusercontent.com/44473782/160729779-41fe207c-b5aa-4fed-
 ## Table of Contents
 
 - [Fixing errors](#Fixing-errors)
+- [Others]
```
Comment (port19x) [inline README.md:36]: The others section should be at the bottom of the table of contents, but above contribution guidelines

File: README.md [hunk 2]
```diff
@@ -33,6 +33,8 @@ https://user-images.githubusercontent.com/44473782/160729779-41fe207c-b5aa-4fed-
...
+   -[mov-cli](https://github.com/mov-cli/mov-cli)
```
Comment (port19x) [inline README.md:37]: This should include much more of our neighbourhood.
A few to consider:
https://github.com/iamchokerman/ani-sync
https://github.com/justfoolingaround/animdl
https://github.com/saikou-app/saikou
https://github.com/sdaqo/anipy-cli
https://github.com/CoolnsX/dra-cla
https://github.com/Soviena/kaa.si-cli
https://github.com/MangDL/MangDL

**Review by port19x — CHANGES_REQUESTED** [review]

**Comment by port19x** [comment]
I'm generally very sceptical, I think chokerman can better judge if this should be a thing

**Comment by iamchokerman** [comment]
i am in favor of this, however as port said it should be at the bottom of the table of contents, but above contribution guidelines, and you need to make the formatting look more presentable

https://github.com/7USTIN/manga-cli

**Comment by FireTonnor** [comment]
Ok I will edit it we I get home

**Review by iamchokerman — APPROVED** [review]

**Review by port19x — APPROVED** [review]

**Pull request merged**